error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/StringRulesTestInput.java

```diff
@@ -39,6 +39,8 @@ ImmutableSet<Boolean> testStringIsNullOrEmpty() {
         getClass().getName() != null && !getClass().getName().isEmpty());
   }
 
+  // XXX: Consider updating qualifier `Predicate#not` if/when conditional `ImportPolicy` is
```

Comment:
> Allow more control over which methods are statically imported by @UseImportPolicy. Sometimes the @AfterTemplate contains more than one static method invocation, and only a subset should be statically imported.

See [here](https://github.com/PicnicSupermarket/error-prone-support/blob/master/error-prone-contrib/README.md#refaster-extension-ideas)

Reply:
Check. Effectively we won't do this, I think. Instead we'll just rely on the existing [`StaticImport`](https://github.com/PicnicSupermarket/error-prone-support/blob/ae22e0ec5e48dba12d93627058c2fd71150c9ff0/error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/StaticImport.java#L53) check.